docs/diagrams/favourite/FavouriteSequenceDiagram.puml

```diff
@@ -0,0 +1,78 @@
+@startuml
+!include ../style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":FavouriteCommandParser" as FavouriteCommandParser LOGIC_COLOR
+participant "f:FavouriteCommand" as FavouriteCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant "pl:List" as PersonList MODEL_COLOR
+end box
+[-> LogicManager : execute(fav 3)
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand(fav)
```

Comment:
```suggestion
LogicManager -> AddressBookParser : parseCommand(fav 3)
```
I think the parameter for the parseCommand should be "fav 3" instead of just "fav"?

Reply:
You're right! Thanks for catching this 👍